heartpoints.sh

```diff
@@ -639,34 +756,47 @@ file_does_not_exist() { local possibleFilePath=$1
 
 gcloud_install() {
     if command_does_not_exist "gcloud"; then
-        brew cask install google-cloud-sdk
+        hp_brew cask install google-cloud-sdk
     fi
 }
 
-gcloud_cli() { local args=$@
-    gcloud_install   
-    gcloud "$@"
+#todo: make sure it runs in docker when things are not mac / brew based
+#by just making sure tools know, in docker, just assume and run command;
+#only autoinstall on mac. (maybe check isMac)
+
+hp_isMac() {
+    strings_are_equal "$(uname)" "Darwin" > /dev/null
 }
 
+# todo: use the locally installed gcloud cask
+# gcloud_cli() { local args=$@
+#     gcloud_install   
+#     gcloud "$@"
+# }
+
 git_currentSha() {
-    echo "$(git rev-parse HEAD)"
+    echo "$(hp_git rev-parse HEAD)"
 }
 
 git_working_directory_is_clean() {
-    [ -z "$(git status --porcelain)" ]
+    [ -z "$(hp_git status --porcelain)" ]
 }
 
-kubectl_install() {
-    brew_install "kubernetes-cli"
+hp_kubectl() { local args="$@"
+    brew_package_run kubectl "$@"
 }
 
-minikube_install() {
-    kubectl_install
-    virtualbox_install
-    if command_does_not_exist minikube; then
-        curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.32.0/minikube-darwin-amd64 && chmod +x minikube && sudo cp minikube /usr/local/bin/ && rm minikube
-    fi
-}
+# kubectl_install() {
+#     brew_install "kubernetes-cli"
+# }
+
+# minikube_install() {
```

Comment:
should this (and `kubectl_install`) remain commented out?

Reply:
i think there may be something to still do here let me check